components/navigation/EventFilter.tsx

```diff
@@ -0,0 +1,61 @@
+import moment from 'moment';
+import React, { useEffect, useState } from 'react';
+
+import { getEvents } from '../../utils/staticHelpers';
+
+interface Event {
+  date: moment.Moment;
```

Comment:
are you sure that the date is of `Moment` type?

Reply:
yes